src/proto/api/v1/transferable_account.proto

```diff
@@ -93,6 +93,7 @@ message IsAccountLockedRequest {
 message CreateAccountResponse {
     string tx_hash = 1;
     string account_id = 2;
+    string address = 3;
```

Comment:
Would `Ethereum Address` be better?

Reply:
Thank you!
I renamed to ethereum_address.